TFlippy_TerritoryControl_Characters_Dev/Entities/NPC/Bison/Bison.as

```diff
@@ -111,6 +111,20 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 			}
 		}
 	}
+	if (cmd == this.getCommandID("write"))
+	{
+		if (isServer())
+		{
+			Random@ rand = Random(this.getNetworkID());
+			CBlob @caller = getBlobByNetworkID(params.read_u16());
```

Comment:
please do a null check (and for all the other times you do this), the chances some data getting corrupted/bit flipped can and will happen at least once

Reply:
Null checks added